pkg/endpoint/bpf.go

```diff
@@ -387,7 +387,14 @@ func (e *Endpoint) regenerateBPF(owner Owner, currentDir, nextDir string, regenC
 	// pre-existing connections using that IP are now invalid.
 	if !e.ctCleaned {
 		go func() {
-			e.scrubIPsInConntrackTable()
+			ipv4 := !option.Config.IPv4Disabled
```

Comment:
That doesn't look like "logging fixes" to me. Move those changes into a different PR?

Reply:
Yeah, arguably that final commit could be in another PR. I can call it out in the description.